compclub/website/templates/website/index.html

```diff
@@ -34,7 +34,7 @@
                 <a class="nav-link" href="Event.html">Event</a>
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="Login.html">Login</a>
+                <a class="nav-link" href="/login">Login</a>
```

Comment:
Let's make it so that when the user is already logged in, this link is `logout` instead. This page has an example of how you can change HTML based on whether is logged in or not: https://docs.djangoproject.com/en/2.0/topics/auth/default/#all-authentication-views.